lesson03/assignment/css/style.scss

```diff
@@ -4,3 +4,67 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+// Below are two mixins that define stylings for flex options. 
+//This helps to minimize rewriting the same code over and over.
+@mixin flexRow {
+    display:flex;
+    flex-direction: row;
+}
+
+@mixin flexCenter {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.card-container {
+    padding: 3rem 0 3rem 0;
+}
+
+.card {
```

Comment:
Using a percentage width here breaks that layout on mobile.  You might try using a value like width: 50rem, and then have a media query to change the flex-direction to column to stack the image and the info when the screen gets smaller.

Reply:
perfect. thank you.